src/domains/stac.ts

```diff
@@ -87,12 +88,13 @@ const thumbnailAssets = (concept: Collection | Granule) => {
     )
     .reduce((metadataAssets, relatedUrl, idx, available) => {
       const [entry, display] = available.length > 1 ? [`_${idx}`, ` [${idx}]`] : ["", ""];
-
+      const relatedUrlKey = extractAssetMapKey(relatedUrl.url);
       const thumbnailAsset: AssetLinks = {};
       thumbnailAsset[`thumbnail${entry}`] = {
         href: relatedUrl.url,
         title: `Thumbnail${display}`,
         description: relatedUrl.description,
+        [relatedUrlKey]: relatedUrl.url,
```

Comment:
Should be sure to update one of the granule files to add a related url to catch this in tests.

Reply:
Updated respective tests